content/03-reference/01-tools-and-interfaces/02-prisma-client/05-filtering.mdx

```diff
@@ -0,0 +1,323 @@
+      some: {
+        published: true,
+      },
+    },
+  },
+})
+```
+
+The following queries support filtering:
+
+- `findMany`
+- `updateMany`
+- `deleteMany`
+
+## Filter conditions
+
+Prisma Client supports the following filter conditions:
+
+| Operator     | Notes                                                |
+| ------------ | ---------------------------------------------------- |
+| `equals`     |                                                      |
+| `not`        | Does not equal.                                      |
+| `in`         | Exists in list.                                      |
+| `notIn`      | Does not exist in list.                              |
+| `lt`         | Less than X.                                         |
+| `lte`        | Less than or equal to X.                             |
+| `gt`         | Greater than X.                                      |
+| `gte`        | Greater than or equal to                             |
+| `contains`   | Does not support wildcards - for example, `b[a-c]d`. |
+| `startsWith` |                                                      |
+| `endsWith`   |                                                      |
+
+### Examples
+
+Retrieve all `Post` records where `title` starts with `Pr` (such as `Prisma`):
+
+```javascript
+const result = await prisma.post.findMany({
+  where: {
+    title: {
+      startsWith: 'Pr',
+    },
+  },
+})
+```
+
+Retrieve all `User` records where `email` ends with `prisma.io`:
+
+```javascript
+const result = await prisma.user.findMany({
+  where: {
+    email: {
+      endsWith: 'prisma.io',
+    },
+  },
+})
+```
+
+Retrieve all `Post` records where `content` contains `databases`:
+
+```javascript
+const result = await prisma.post.findMany({
+  where: {
+    content: {
+      contains: 'databases',
+    },
+  },
+})
+```
+
+Retrieve all `Post` records where `date_created` is greater than March 19th, 2020:
+
+```javascript
+const user = await prisma.post.findMany({
+  where: {
+    date_created: {
+      gte: new Date('2020-03-19T14:21:00+0200') /* Includes time offset for UTC */,
+    },
+  },
+})
+```
+
+## Combine multiple filter conditions
+
+You can combine multiple filter conditions with the following operators:
+
+| Operator | Notes                                    |
+| -------- | ---------------------------------------- |
+| `AND`    | All conditions must return true.         |
+| `NOT`    | All conditions must return false.        |
+| `OR`     | One or more conditions must return true. |
+
+### Examples
+
+Retrieve all `Post` records where the `content` field contains `Prisma` and `published` is `false`:
+
+```javascript
+const result = await prisma.post.findMany({
+  where: {
+    AND: [
+      {
+        content: {
+          contains: 'Prisma',
+        },
+      },
+      {
+        published: {
+          equals: false,
+        },
+      },
+    ],
+  },
+})
+```
+
+Note that you can write the same query without `AND`:
```

Comment:
I feel like it might be worth pointing that out explicitly under the operator table for `AND`, `OR` and `NOT` instead of just mentioning it in the context of a concrete example.

Reply:
Done! ✅